interfaces/webrtc-stats.idl

```diff
@@ -277,7 +277,6 @@ dictionary RTCIceCandidateStats : RTCStats {
              long priority;
              DOMString url;
              DOMString relayProtocol;
-             boolean deleted = false;
```

Comment:
This is a real change, but dictionaries aren't tested at all so it doesn't affect results.